src/OpenConext/Profile/Value/Consent.php

```diff
@@ -95,4 +113,12 @@ public function getConsentType()
     {
         return $this->consentType;
     }
+
+    /**
+     * @return null|AttributeSet
+     */
+    public function getAttributes()
+    {
+        return $this->attributes;
```

Comment:
Let's refrain from using nulls like all other places (ie. has-er). Also, maybe releasedAttributes or attributesReleased more clearly describes what the attributes mean in the context of a consent.